### DISCUSSION TITLE
Objets en récompenses de missions

### DISCUSSION THREAD

**Discussion by OrionYp** [Crownicles/Crownicles#2156] [Nouvelles idées]
@Je propose : 
1) d'ajouter des items en récompenses de certaines missions de campagne, les missions quotidiennes, et missions secondaires 
2) les items obtenus en récompenses de missions de campagne pourraient être prédéfinis pour qu'un joueur qui ait fini la campagne (C'est sensé être long) puisse avoir un équipement correct.

**Comment by K-w12** [comment]
Bonne idée
